src/PluginClient.php

```diff
@@ -127,7 +128,16 @@ private function createPluginChain($pluginList, callable $clientCallable)
     {
         $firstCallable = $lastCallable = $clientCallable;
 
-        while ($plugin = array_pop($pluginList)) {
+        /*
+         * Inject debug plugins between each plugin.
+         */
+        $pluginListWithDegug = $this->options['debug_plugins'];
```

Comment:
I dont know whats wrong with me. I've corrected this typo at least 10 times... 

Thank you.